src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static String getRandomColor() {
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        int randomIndex = random.nextInt(colors.length);
+        return colors[randomIndex].toString();
```

Comment:
Use name() for getting String representation of enum constants instead of toString(). The name() method is designed to return the exact name of the enum constant, which is what you typically want to use.

Reply:
https://github.com/PavloMelnyk10/jv-lottery/blob/master/checklist.md

Reply:
see [common mistakes file](https://github.com/PavloMelnyk10/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)
use name()